City-of-Pori/pori-kada
# PORI-211 Fixed filter issue with Firefox

To test:

1. drush cc all
2. Go to https://local.visitpori.fi/nae-ja-koe/nahtavyydet using firefox and make sure the filtering works.
